plugins/ads.js

```
@@ -2,38 +2,33 @@ import { bot } from '#lib';
 
-		for (const groupId of groupIds) {
-			await delay(1500);
-			await message.send(broadcastMessage, {
-				jid: groupId,
-				contextInfo: messageOptions,
-			});
-		}
-
-		return await message.send('```Shared to ' + groupIds.length + ' Groups.```');
-	},
+    return await message.send('```Shared to ' + groupIds.length + ' Groups.```');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)